src/components/autocomplete/js/autocompleteController.js

```diff
@@ -660,21 +660,35 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
 
   /**
    * Determines if the menu should be hidden.
-   * @returns {boolean}
+   * @returns {boolean} true if the menu should be hidden
    */
   function shouldHide () {
-    if (!isSearchable()) return true;    // Hide when not able to query
-    else return !shouldShow();            // Hide when the dropdown is not able to show.
+    // Hide when not able to run the query.
```

Comment:
The fact that this isn't just `return !this.shouldShow()` feels kind of weird to me

Reply:
I agree. I've made an update. What do you think?